src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -350,6 +350,87 @@ private void SetInputPullModePi4(int pinNumber, PinMode mode)
             }
         }
 
+        /// <inheritdoc />
+        protected internal override void SetAlternatePinMode(int pinNumber, AlternatePinMode altMode)

```

Comment:
I know it's internal but I haven't seen anywhere explicit where the pinNumber is checked to be valid. I most likely have missed it. So if you're sure it's checked, ignore this comment :-)

Reply:
I'll check again, but generally the pin number is checked by the GpioController.

Reply:
You were right, I had missed the test.